src/molecules/formfields/DateField/index.js

```diff
@@ -52,9 +52,13 @@ DateField.propTypes = {
   fieldRef: PropTypes.func,
 
   /**
-   * Adds a tooltip to the label. Provide string for text to be placed inside tooltip popup
+   * either a handler for clicking the tooltip, or text to go in the tooltip for the label
    */
-  tooltip: PropTypes.string,
+  tooltip: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.string,
+    PropTypes.object
```

Comment:
Could you explain when the `tooltip` would be an object?

Reply:
yeah in the life-web PR you can see some examples but some tooltips were passed something that looks like this:
`{ children: "Don't include income from a spouse or partner", revealOnClick: true }`
in case other teams use tooltips in that same way, I just want to make sure we preserve this for them

Reply:
oh, I see that now. Thanks!